.github/workflows/continuous-delivery.yml

```diff
@@ -1157,22 +1158,13 @@ jobs:
       -
         name: Teardown AKS shared resources
         if: always()
-        run: |
-          az account set --subscription ${{ secrets.AZURE_SUBSCRIPTION }}
-          attempt=1
-          max_attempts=3
-          while [ "${attempt}" -le "${max_attempts}" ]; do
-              echo "Deleting storage account. Attempt ${attempt} of ${max_attempts}"
-              az storage account delete -y --resource-group ${{ secrets.AZURE_RESOURCEGROUP }} --name ${{ env.AZURE_STORAGE_ACCOUNT }}
-              status=$?
-              if [[ $status == 0 ]]; then
-                echo "Storage account deleted"
-                break
-              fi
-              echo "Failed deleting storage account ${{ env.AZURE_STORAGE_ACCOUNT }}, retrying"
-              sleep 5
-              attempt=$((attempt+1))
-          done
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 1
```

Comment:
Are you sure that this timeout is enough? The previous code did not constrain the execution time.

Reply:
That code runs in about ~10 seconds (looking at some recent E2Es). I've increased the max timeout to 5 minutes though, which should be more than enough